playbook.yaml

```diff
@@ -0,0 +1,51 @@
+---
+- hosts: localhost
+  connection: local
+  gather_facts: yes
+  roles:
+  - role: 'grycap.docker'
+
+- hosts: localhost
+  connection: local
+  gather_facts: yes
+  tasks:
+  - name: Get dashboard repo
+    ansible.builtin.git:
+      repo: 'https://github.com/enolfc/horizon-aggregator.git'
```

Comment:
Should this go somewhere under https://github.com/EGI-Federation/?

Reply:
yes, or even just have one repo if we feel more appropriate